### ISSUE TITLE
Can't install xpi after converting it from crx.

### ISSUE THREAD

**Issue by Dr-Sauce** [Noitidart/Chrome-Store-Foxified#149]
I managed to convert crx to xpi using Chrome store foxified. But I can't install it because of Firefox privacy problems. Is there a way to install xpi's?

![20210415_105414](https://user-images.githubusercontent.com/82555878/114804516-5b983600-9ddc-11eb-8432-a20be0eb79bd.png)

**Comment by grysor** [comment]
> I managed to convert crx to xpi using Chrome store foxified. But I can't install it because of Firefox privacy problems. Is there a way to install xpi's?
> 
> ![20210415_105414](https://user-images.githubusercontent.com/82555878/114804516-5b983600-9ddc-11eb-8432-a20be0eb79bd.png)







Hello :)  
i'm french sorry for my bad english i use google translate 
Me i just managed to temporarily install this chrome extension in my Firefox 88.0 :

https://chrome.google.com/webstore/detail/cookiesblock-cookie-pop-u/ajkknbgennjgacpfbhdobipfhhikbldg

I first tried by Chrome store foxified clicking on " add to firefox " And i got the message " Parsing " 
Then i tried to install it in .crx format and it did not work away
Then i converted it to xpi by Chrome store foxified  ( if the conversion does not work, click on "Clear Memory" in the "settings " of Chrome store foxified , then disable extension Chrome store foxified and reactivate it )

Then you have to type in the Firefox URL address bar : 
" about:debugging "
Then click on :
" Load Temporary Add-on " 
and my XPI extension has been installed but temporarily :

<a href="https://goopics.net/i/JJZyX"><img src="https://i.goopics.net/JJZyX.jpg" alt="Image"></a>

unfortunately as it is temporary, closing firefox and then re-opening it, the extension has disappeared

If my testimony can help .....

**Comment by Dr-Sauce** [comment]
Thanks for your help~!

**Comment by e-t-l** [comment]
@Dr-Sauce after creating the extension, you have to click "Sign Now" then log into your Mozilla AMO account (or create one for free if you haven't yet).
 
That may still fail to upload/install in Firefox Portable, though, so best practice is to download unsigned, then upload it to your AMO Developer Hub at [https://addons.mozilla.org/en-US/developers/addon/submit/upload-unlisted](https://addons.mozilla.org/en-US/developers/addon/submit/upload-unlisted). It'll get processed, then you'll receive an email. If you click on the version name in the link that opens, there will then be a hyperlink to an .xpi that will install the addon when clicked.

**Comment by tastyratz** [comment]
Did that ever end up working for you?

I'd kill for Cluster tab manager in my firefox and I've been trying everything I could to get this going.

**Comment by Dr-Sauce** [comment]
@tastyratz Yep, you can install a extension by sideloading it. (You can only use the extension temporarily.)

Here's how to install a Chrome extension on Firefox without **Chrome Store Foxified**.
- - - - -
First install this extension on Chrome. 

https://chrome.google.com/webstore/detail/crx-extractordownloader/ajkhmmldknmfjnmeedkbkkojgobmljda
![0001](https://user-images.githubusercontent.com/82555878/153746410-69e50749-246a-4dba-b461-db79e29dea29.png)

And then open the extension link you want to install on FireFox.

Ex) Cluster tab manager: https://chrome.google.com/webstore/detail/cluster-window-tab-manage/aadahadfdmiibmdhfmpbeeebejmjnkef
![0002](https://user-images.githubusercontent.com/82555878/153746435-975e4e7a-dcc4-4deb-9e81-26372aec5b36.png)

Open **CRX Extractor/Downloader** from the extension tab. 

Press **Download as ZIP**.
![0003](https://user-images.githubusercontent.com/82555878/153746439-e22f911d-0612-4a43-b4f4-27132fd90884.png)

Now open `about:debugging#/runtime/this-firefox` in firefox. 

And press **Load Temporary Add-on**

![0004](https://user-images.githubusercontent.com/82555878/153746609-16cb3729-d398-47d0-8956-30dc8de1d0bc.png)

Select the zip you downloaded from Chrome.

![0005](https://user-images.githubusercontent.com/82555878/153746607-d92de185-6664-4705-b59e-0ae0bb3ae95b.png)

And your extension will be installed.

(+ Seems like, Cluster tab manager doesn't work properly on FireFox.)

![0006](https://user-images.githubusercontent.com/82555878/153746644-11a13ffd-62e9-465b-8937-5c5a431e84ed.png)

**Comment by FoxRefire** [comment]
#165
